<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>971</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>961</width>
        <height>541</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Colorful Modern Tropical Fruits Blogger Summer Zoom Virtual Background  (2).png</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="PASSWORD">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>390</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#PASSWORD{
	background-color: rgb(255, 255, 0);
	border-radius: 5px;
	color: rgb(0, 0, 0);
}

QLineEdit#PASSWORD:hover{
	background-color: rgb(255, 241, 138);
	border-radius: 5px;
	color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="USERNAME">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>330</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#USERNAME{
	background-color: rgb(255, 255, 0);
	border-radius: 5px;
	color: rgb(0, 0, 0);
}

QLineEdit#USERNAME:hover{
	background-color: rgb(255, 241, 138);
	border-radius: 5px;
	color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="MAIN_LOGIN">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>470</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#MAIN_LOGIN{
	background-color: rgb(255, 255, 255);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}

QPushButton#MAIN_LOGIN:hover{
	background-color: rgb(255, 241, 138);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>LOGIN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MAIN_CLOSE">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>470</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#MAIN_CLOSE{
	background-color: rgb(255, 255, 255);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}

QPushButton#MAIN_CLOSE:hover{
	background-color: rgb(255, 241, 138);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>CLOSE</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="Background2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>971</width>
        <height>551</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>2.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="FRUIT_CAPTURE">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>591</width>
        <height>391</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="FRUIT_SIZE">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>410</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#FACE_NAME{
	background-color: rgb(255, 255, 255);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}

QLineEdit#FACE_NAME:hover{
	background-color: rgb(255, 241, 138);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="FRUIT_CATEGORY">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>410</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#FACE_NAME{
	background-color: rgb(255, 255, 255);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}

QLineEdit#FACE_NAME:hover{
	background-color: rgb(255, 241, 138);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="FRUIT_NAME">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat SemiBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#FACE_NAME{
	background-color: rgb(255, 255, 255);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}

QLineEdit#FACE_NAME:hover{
	background-color: rgb(255, 241, 138);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="DETECT">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>460</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#DETECT{
	background-color: rgb(255, 255, 255);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}

QPushButton#DETECT:hover{
	background-color: rgb(255, 241, 138);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>DETECT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="STOP">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>460</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#DETECT{
	background-color: rgb(255, 255, 255);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}

QPushButton#DETECT:hover{
	background-color: rgb(255, 241, 138);
	border-radius: 2px;
	color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
